debian/control

```diff
@@ -70,6 +70,7 @@ Vcs-Browser: https://salsa.debian.org/gnome-team/gnome-shell
 Package: gnome-shell
 Architecture: linux-any
 Depends: adwaita-icon-theme,
+         dconf-cli,
```

Comment:
Can this be pushed up to Debian, presumably that's where the postinst script comes from?

Reply:
I guess you removed this instead?

Reply:
No, the commit is still in the branch.

Reply:
I see, it was in a collapsed section.